OMCompiler/Compiler/SimCode/SimCodeUtil.mo

```diff
@@ -580,6 +580,23 @@ algorithm
     // generate cref2simVar hash table
     crefToSimVarHT := createCrefToSimVarHT(modelInfo);
 
+    // check for alias variables Variabilty with look up to actual variable
```

Comment:
We basically do the same thing here from line 584 to line 600. Write a function returning a tuple instead to avoid duplication : )